handlers/reader.go

```diff
@@ -34,11 +34,12 @@ func getServiceList(functionNamespace string, clientset *kubernetes.Clientset) (
 
 		labels := item.Spec.Template.Labels
 		function := requests.Function{
-			Name:            item.Name,
-			Replicas:        replicas,
-			Image:           item.Spec.Template.Spec.Containers[0].Image,
-			InvocationCount: 0,
-			Labels:          &labels,
+			Name:              item.Name,
+			Replicas:          replicas,
+			Image:             item.Spec.Template.Spec.Containers[0].Image,
+			AvailableReplicas: int(item.Status.ReadyReplicas),
```

Comment:
Shouldn't `item.Status.AvailableReplicas` be used for this? (as per [k8s docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/))
Looks like a perfect match!

Reply:
@stefanprodan please could you comment?

Reply:
I can't see `ReadyReplicas` in the docs link but I can see `AvailableReplica` - we should clarify which is the one we need for a container that has passed its readiness/liveness probe.

Reply:
The `ReadyReplicas` will target all ReplicaSets during a rolling update, while  `AvailableReplicas` tracks the desired replica count in the active RS after the min ready seconds have passed. I would use `AvailableReplicas`.